Formula/pngquant.rb

```diff
@@ -13,14 +14,14 @@ class Pngquant < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "rust" => :build
   depends_on "libpng"
   depends_on "little-cms2"
 
   def install
-    system "cargo", "build", "--release"
-    bin.install "target/release/pngquant"
-    man1.install "pngquant.1"
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "install"
+    lib.install "lib/libimagequant.a"
+    include.install "lib/libimagequant.h"
```

Comment:
Ditto.

Reply:
The whole point of this pull request is that `make install` for libquant _does not_ install the library or header file.

Reply:
That is asking us to override upstream's judgment about what should be installed. The proper way to handle this is for us to defer to `make install` and for you to ask upstream to include the include file(s).

Reply:
I must be missing a hidden special doc that says the mission of Homebrew is to enforce software packaging standards on the world. Feel free to take the pull request or leave it.

Reply:
The issue is not about enforcing packaging standards but about making sure the maintenance of the formula is sustainable and reflects upstream's wishes. For example, let's say that upstream adds more header files or another static library. Then someone would need to notice that happened and change the install method. That's very brittle. If instead we use `make install`, and upstream wants to start installing more include files or additional static libraries, then that takes care of itself automatically because they'll update the `Makefile` accordingly. Just adding manual installations doesn't leave the formula in a sustainable state.

We're willing to install some things like an include file manually temporarily, but only if you've also done the leg work to make sure that's not a permanent situation by at least requesting upstream make that tweak to their Makefile. Filing that issue would take you less time than you've spent expressing your displeasure with how we run Homebrew.